lib/reporters/error-reporter.js

```diff
@@ -0,0 +1,37 @@
+const Logger = require('../logger');
+const colors = require('colors');
+const path = require('path');
+const isArray = require('lodash/isArray');
+
+const logger = new Logger();
```

Comment:
Throughout the whole app the `logger` is needed because `console.log` is completely muted.